cypress/support/fragments/inventory/inventoryInstance.js

```diff
@@ -617,6 +620,10 @@ export default {
   clickAddItemByHoldingName(holdingName) {
     const holdingSection = section.find(Accordion(including(holdingName)));
     cy.do(holdingSection.find(addItemButton).click());
+
+    ItemRecordEdit.waitLoading('');
```

Comment:
This should be Instance resourse_title name not empty string

Reply:
Done